server/src/controllers/users.js

```diff
@@ -21,7 +39,7 @@ exports.addUser = (req, res) => {
   const { username, password } = req.body;
 
   getUserByName(username, (err, users) => {
-    if (err) res.json(err.message);
+    if (err) console.log(err.message);
```

Comment:
` if (err) console.log(err.message);`
same here, this is not proper error handling